src/main/java/control/TimelineController.java

```diff
@@ -267,7 +281,30 @@ public void load() {
                 alert.setContentText("The format in the selected file was not recognized");
                 alert.showAndWait();
             } else {
-                project = temp;
+                controllerManager.setScriptingProject(temp);
+                project = controllerManager.getScriptingProject();
+                addLoadedBlocks(project);
+                numTimelines = project.getCameraTimelines().size();
+            }
+        }
+    }
+    
+    /**
+     * Add the blocks that were loaded from file to the UI.
+     * @param project the project that was loaded
+     */
+    private void addLoadedBlocks(ScriptingProject project) {
+        System.out.println("Adding loaded blocks");
+        for (int i = 0; i < project.getCameraTimelines().size();i++) {
+            System.out.println("Outer loop");
+            CameraTimeline timeline = project.getCameraTimelines().get(i);
+            int amountShots = timeline.getShots().size();
+            for (int j = 0; j < amountShots;j++) {
+                System.out.println(timeline.getShots().size());
```

Comment:
Sorry, forgot to remove them. I won't replace them all with log.debug, just a single log statement at the start of the method